stable/chat/v1/messages.proto

```diff
@@ -537,8 +513,11 @@ message TriggerActionRequest {
   uint64 channel_id = 2;
   // Message ID of the message you want to trigger an action in.
   uint64 message_id = 3;
+  // Data that alllows the bot to discern which action the user is trying to trigger.
+  // A server MUST verify that this info refers to a valid action for the target message.
```

Comment:
isn't bytes too arbritrary for a trigger action request? wouldn't it be more appropriate to have data be properly defined for trigger requests?

Reply:
bots define valid input as bytes; for the text input action this doesn't work, might be worth adding an oneof there